### PR TITLE
Add a fake slug for websites (bug 1174931)

### PR DESCRIPTION
**Pull request by diox**

### CODE REVIEW

**Comment by mstriemer** [comment]
Do we not have a fireplace serializer? I thought we only wanted to expose this to the fireplace versions of our API?

r+wc

**Comment by diox** [comment]
I realized when writing the patch that we don't have it yet, no - we could though, I'll make one and update this patch.

**Comment by diox** [comment]
@mstriemer what fields does fireplace not need in this list ? : `['categories', 'description', 'device_types', 'icons', 'id', 'keywords', 'mobile_url', 'name', 'short_name', 'slug', 'title', 'url']`

Hopefully only `title` ? Wanted to check with you first :)

**Comment by mstriemer** [comment]
Looks like we don't use `title`, `keywords` and `mobile_url`. I'm not sure if that's going to change.

File: mkt/websites/serializers.py [hunk 1]
```diff
@@ -26,8 +26,8 @@ class Meta:
                   'url']
 
     def get_icons(self, obj):
-        return dict([(icon_size, obj.get_icon_url(icon_size))
-                     for icon_size in CONTENT_ICON_SIZES])
+        return {icon_size: obj.get_icon_url(icon_size)
```
Comment (mstriemer) [inline mkt/websites/serializers.py]: :clap::clap::clap:

**Comment by mstriemer** [comment]
lgtm and tests pass.

File: mkt/fireplace/tests/test_views.py [hunk 2]
```diff
@@ -146,16 +157,49 @@ def test_icons(self):
...
-    # The fireplace variant of multi-search view is only different for apps
-    # at the moment, not websites, so simply re-use TestSearchView tests.
+class TestMultiSearchView(RestOAuth, ESTestCase):
+    fixtures = fixture('user_2519', 'webapp_337141')
+
     def setUp(self):
         super(TestMultiSearchView, self).setUp()
         self.url = reverse('fireplace-multi-search-api')
+        self.website = website_factory()
+        self.website.popularity.add(WebsitePopularity(region=0, value=666))
```
Comment (robhudson) [inline mkt/fireplace/tests/test_views.py]: :imp:

**Pull request merged**